src/firestore/firekit.ts

```diff
@@ -1743,6 +1751,42 @@ export class RoarFirekit {
     return task;
   }
 
+  async updateTaskOrVariant(updateData: UpdateTaskVariantData) {
+    try {
+      this._verifyAuthentication();
+    } catch (error) {
+      console.log('auth error: ', error);
+    }
+
+    try {
+      this._verifyAdmin();
+    } catch (error) {
+      console.log('admin error: ', error);
+    }
```

Comment:
Why do this as apposed to
```
this._verifyAdmin()
```

`_verifyAdmin()` contains a `_verifyAuthentication` call so one implies the other.

Reply:
And I'd like to avoid the `try`/`catch` so that we don't swallow the auth or admin errors.